### DISCUSSION TITLE
Geolocation NEWS

### DISCUSSION THREAD

**Discussion by 97-42** [SethBurkart123/EvenBetterSEQTA#144] [Ideas]
I am curious as to what the geolocation analytics are on the chrome web store. If enough users use BS+ overseas (for example Europe or the USA) I believe it would be a good idea to give them News sources from their own country as I doubt the entire world is hooked on aussie news.

**Comment by bmeehankas** [comment]
We use BS+ in Kaohsiung, Taiwan. It's a big hit with our students. We sure would appreciate being able to select a non-Australian news source... though it does keep me up-to-date when I chat with the devs at Education Horizons in Perth.

I've even heard rumblings of some of our students working on a theme file for our school!

**Reply by SethBurkart123** [reply]
Yes I'll definitely have to consider it. @ar-cyber has opened #196 so I'll have a look at what news sources I can get, if you do have any requests or recommendations I am willing to consider whichever sources everyone would be interested in.

**Comment by ar-cyber** [comment]
>  I believe it would be a good idea to give them News sources from their own country as I doubt the entire world is hooked on aussie news.

~~I feel like the issue with this is that we'd have to database 198 different news sites for every country. It may, however, be something that can be done gradually over time.~~

NEVERMIND. This isn't actually quite hard to implement.
